nuxeo-dm/nuxeo-platform-default-config/src/main/resources/OSGI-INF/document-pageprovider-contrib.xml

```diff
@@ -67,6 +67,47 @@
       <pageSize>20</pageSize>
     </coreQueryPageProvider>
 
+    <coreQueryPageProvider name="advanced_document_trash_content">
+      <trackUsage>true</trackUsage>
+      <property name="maxResults">DEFAULT_NAVIGATION_RESULTS</property>
+      <whereClause docType="AdvancedContent">
+        <fixedPart>
+          ecm:isCheckedInVersion = 0 AND
+          ecm:mixinType !=
+          'HiddenInNavigation' AND ecm:isTrashed = 1
```

Comment:
`ecm:isTrashed = 1` -> `ecm:currentLifeCycleState = 'deleted'` for 9.10

Reply:
This is preoccupying, this means this is not tested... or the Jenkins job wouldn't have succeeded.